docs/error-handling-guide.md

```
@@ -0,0 +1,350 @@
+Shows recent connection issues:
+
+```
+Recent Connection Issues:
+• SSE connection timeout after 15 seconds
+• VANA backend returned 503 Service Unavailable
+```
+
+### Agent Progress
+Enhanced progress display with error states:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)